<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>SinCleaner</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">

    <!-- Tabs -->
    <item>
     <widget class="QTabWidget" name="tabWidget">

      <!-- FILE TAB -->
      <widget class="QWidget" name="tabFile">
       <attribute name="title">
        <string>File</string>
       </attribute>
       <layout class="QVBoxLayout">

        <item>
         <widget class="QPushButton" name="btnSelectFile">
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>Choose File</string>
          </property>
         </widget>
        </item>

        <item>
         <widget class="QLabel">
          <property name="text">
           <string>Naming style:</string>
          </property>
         </widget>
        </item>

        <item>
         <widget class="QComboBox" name="comboNamingStyle">
          <item><property name="text"><string>camelCase</string></property></item>
          <item><property name="text"><string>snake_case</string></property></item>
          <item><property name="text"><string>kebab-case</string></property></item>
          <item><property name="text"><string>PascalCase</string></property></item>
          <item><property name="text"><string>UPPER_CASE</string></property></item>
         </widget>
        </item>

        <item>
         <widget class="QPushButton" name="btnRenameFile">
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>Rename file</string>
          </property>
         </widget>
        </item>

        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>

       </layout>
      </widget>

      <!-- INFO TAB -->
      <widget class="QWidget" name="tabInfo">
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel">
          <property name="text">
           <string>File information:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textInfo"/>
        </item>
       </layout>
      </widget>

      <!-- RESIZE TAB -->
      <widget class="QWidget" name="tabResize">
       <attribute name="title">
        <string>Resize</string>
       </attribute>
       <layout class="QVBoxLayout">

        <item>
         <layout class="QFormLayout">
          <item row="0" column="0">
           <widget class="QLabel">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinWidth">
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinHeight">
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>

        <item>
         <widget class="QPushButton" name="btnResize">
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>Resize Image</string>
          </property>
         </widget>
        </item>

        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>

       </layout>
      </widget>

      <!-- CONVERT TAB -->
      <widget class="QWidget" name="tabConvert">
       <attribute name="title">
        <string>Convert</string>
       </attribute>
       <layout class="QVBoxLayout">

        <item>
         <widget class="QLabel">
          <property name="text">
           <string>Target format:</string>
          </property>
         </widget>
        </item>

        <item>
         <widget class="QComboBox" name="comboFormat">
          <item><property name="text"><string>PNG</string></property></item>
          <item><property name="text"><string>JPG</string></property></item>
          <item><property name="text"><string>WEBP</string></property></item>
         </widget>
        </item>

        <item>
         <widget class="QPushButton" name="btnConvert">
          <property name="minimumHeight">
           <number>40</number>
          </property>
          <property name="text">
           <string>Convert File</string>
          </property>
         </widget>
        </item>

        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>

       </layout>
      </widget>

      <!-- METADATA TAB -->
      <widget class="QWidget" name="tabMetadata">
       <attribute name="title">
        <string>Metadata</string>
       </attribute>
       <layout class="QVBoxLayout">

        <item>
         <widget class="QTextEdit" name="textMetadata"/>
        </item>

        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="btnClearMetadata">
            <property name="minimumHeight">
             <number>36</number>
            </property>
            <property name="text">
             <string>Clear metadata</string>
            </property>
           </widget>
          </item>

          <item>
           <widget class="QPushButton" name="btnOpenMetadataFull">
            <property name="minimumHeight">
             <number>36</number>
            </property>
            <property name="text">
             <string>Open full view</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>

       </layout>
      </widget>

     </widget>
    </item>

    <!-- Bottom status -->
    <item>
     <widget class="QLabel" name="labelCurrentFile">
      <property name="text">
       <string>No file selected</string>
      </property>
      <property name="styleSheet">
       <string>color: gray; padding: 6px;</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>

 <resources/>
 <connections/>
</ui>
